apps/web/src/app/(main)/pricing/page.tsx

```diff
@@ -0,0 +1,89 @@
+import AnimatedTab from '@/components/ui/animated-tabs'
+import { ColorBGSVG } from '@public/hero'
+import PriceCard from '@/components/pricing/card'
+
+function About(): React.JSX.Element {
+  const tabsData = [
+    { id: 'monthly', label: 'Monthly' },
+    { id: 'yearly', label: 'Yearly', tag: '-20%', special: true }
+  ]
+
+  const priceCardData = [
+    {
+      title: 'Free',
+      description: 'For hobbyists and beginners',
+      price: 0,
+      isPopular: false,
+      spaceProjects: 1,
+      spaceUsers: 1,
+      spaceLiveSupport: false,
+      miscFeatures: ['Unlimited Users', '1 Workspace']
+    },
+    {
+      title: 'Pro',
+      description: 'For small teams and startups',
+      price: 15,
+      isPopular: true,
+      spaceProjects: 5,
+      spaceUsers: 5,
+      spaceLiveSupport: true,
+      miscFeatures: ['Unlimited Users', '5 Workspaces']
+    },
+    {
+      title: 'Enterprise',
+      description: 'For large teams and enterprises',
+      price: 45,
+      isPopular: false,
+      spaceProjects: 100,
+      spaceUsers: 1000,
+      spaceLiveSupport: true,
+      miscFeatures: ['Unlimited Users', '10 Workspaces']
+    }
+  ]
+
+  return (
+    <div className="relative flex flex-col items-center justify-center ">
+      <ColorBGSVG className="absolute -z-10 h-screen w-screen" />
+
+      <div className="pb-4 pt-14">
+        <h3 className="w-24 rounded-full border-[1px] border-white border-opacity-[.04] bg-white bg-opacity-5 p-2 text-center text-xs tracking-widest text-white md:text-sm">
+          PRICING
+        </h3>
```

Comment:
**Quick tip**
you can use `uppercase` from tailwind to capitalize all texts in a div rather than doing it manually, this is a good practice this will keep the code maintainable.